ui-core/src/components/EditAddress.tsx

```diff
@@ -0,0 +1,220 @@
+import { AddressValidationResult, MailingAddress } from 'src/types/address'
+import classNames from 'classnames'
+import { isNil, sortBy } from 'lodash'
+import React, { useEffect, useState } from 'react'
+import { getAllCountries } from '../i18nUtils'
+import { isAddressFieldValid } from '../addressUtils'
+import CreatableSelect from 'react-select/creatable'
+
+/**
+ *
+ */
+export default function EditAddress(
+  {
+    mailingAddress,
+    setMailingAddress,
+    validationResult,
+    showLabels = true,
+    language = 'en' // todo: accept internationalized labels
```

Comment:
Just filed a ticket for this. Can you attach [JN-910](https://broadworkbench.atlassian.net/browse/JN-910) to this comment?

Reply:
This is very partially completed, I made the navbar `Profile` button internationalized but the form itself isn't.